src/components/ratingsreviews/ReviewCard.jsx

```diff
@@ -1,10 +1,18 @@
 import React, { useState , useEffect} from 'react';
 import { AiFillCheckCircle } from 'react-icons/ai';
 import StarDisplayQuarters from './StarDisplayQuarters.jsx';
+import api from '../../../server/api.js';
 import Thumbnails from './../shared/Thumbnails.jsx'
 
-
 const ReviewCard = ({aReview}) => {
+  const [expand, setExpand] = useState(false);
+  const [clickIndex, setClickIndex] = useState(0)
+  const [imageIndex, setImageIndex] = useState(0);
+  const [isHelpful, setIsHelpful] = useState(0);
+
+  useEffect(() => {
+    aReview.helpfulness ? setIsHelpful(aReview.helpfulness) : isHelpful;
+  }, []);
```

Comment:
When should this useEffect be running, it is not watching anything so it may run more than intented

Reply:
It runs at the component mount to reset - helpfulness if it hasn't been loaded yet b/c its data is more deeply nested.